documents/customer.md

```diff
@@ -0,0 +1,52 @@
+## Customer
+
+### Create customer
+```php
+$api->customer->create(array('name' => 'Razorpay User', 'email' => 'customer@razorpay.com','contact'=>'9123456780','notes'=> array('notes_key_1'=> 'Tea, Earl Grey, Hot','notes_key_2'=> 'Tea, Earl Grey… decaf'));
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| name*          | string      | Name of the customer                        |
+| email        | string      | Email of the customer                       |
+| contact      | string      | Contact number of the customer              |
+| notes         | array      | A key-value pair                            |
+
+-------------------------------------------------------------------------------------------------------
+### Edit customer
+```php
+$api->customer->fetch($customerId)->edit(array('name' => 'Razorpay User', 'email' => 'customer@razorpay.com','notes'=> array('notes_key_1'=> 'Tea, Earl Grey, Hot','notes_key_2'=> 'Tea, Earl Grey… decaf')));
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| customerId*          | string      | The id of the customer to be updated  |
+| email        | string      | Email of the customer                       |
+| contact      | string      | Contact number of the customer              |
+| notes         | array      | A key-value pair                            |
+
+-------------------------------------------------------------------------------------------------------
+
+### Fetch a customer
+```php
+$api->customer->fetch($customerId)
```

Comment:
@HancilSequeira make following changes in all, store response in variable and then talk about how to see API response

$customer = $api->customer->fetch($customerId)

click [here](entityResponse.md#customer-api-response) to view `$customer` attributes

Reply:
@chirag-patel-rp will it not create too much back and fro between files considering for each api for the merchant. Instead of that is ok if I add a response after each API section

Reply:
Yes, that is also fine. I was suggesting this approach to avoid repetitive work

Reply:
Ok, then I will go with 2nd approach where I will add the response after each API